docs/vision-usage.rst

```diff
@@ -142,15 +81,16 @@ attempt to identify those objects.
 Landmark Detection
 ~~~~~~~~~~~~~~~~~~
 
-The API will attemtp to detect landmarks such as Mount Rushmore and
-the Sydney Opera House. The API will also provide their known geographical
-locations if available.
+:meth:`detect_landmarks()` will attempt to detect landmarks such as
+"Mount Rushmore" and the "Sydney Opera House". The API will also provide their
+known geographical locations if available.
 
 .. code-block:: python
 
     >>> from google.cloud import vision
     >>> client = vision.Client()
-    >>> image = client.image('./image.jpg')
+    >>> with open('./image.jpg', 'rb') as image_file:
+    ...     image = client.image(content=image_file.read())
```

Comment:
1. I believe it was in an early PR that got closed.
2. I would like to. I've added it to #2753. 
My current plan is something like `image = client.image(filename='./my.jpg')`.